src/sentry/static/sentry/app/routes.jsx

```diff
@@ -908,6 +908,21 @@ function routes() {
               component={errorHandler(LazyLoad)}
             />
           </Route>
+          <Route
+            path="/organizations/:orgId/incidents/"
+            componentPromise={() =>
+              import(/* webpackChunkName: "OrganizationIncidentsContainer" */ './views/organizationIncidents/index')
+            }
+            component={errorHandler(LazyLoad)}
+          >
+            <IndexRoute
+              componentPromise={() =>
+                import(/* webpackChunkName: "OrganizationIncidents" */ './views/organizationIncidents/list/index')
```

Comment:
Should the chunk name be "OrganizationIncidentsList"?

Reply:
Just copying the naming structure of monitors but I can update to call the chunk list